### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by dinhvareal**

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  color: #000000;
+  text-align: center;
+  background-color: #FFFAF0;
+  a {
+    color: #000000;
+    &:hover {
+      color: #000080;
+    }
+  }
+  ul {
```
Comment (dinhvareal) [inline app/assets/stylesheets/custom.scss]: cho nay ko can thiet

File: app/controllers/homepage_controller.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class HomepageController < ApplicationController
+	def new
```
Comment (hieunguyentrung) [inline app/controllers/homepage_controller.rb]: Use two spaces per indentation level
https://rubystyle.guide/#spaces-indentation

File: app/helpers/homepage_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module HomepageHelper
```
Comment (hieunguyentrung) [inline app/helpers/homepage_helper.rb]: Please delete the file if not used.

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<footer>
+  <div class="row" style="margin-bottom:15px">
```
Comment (hieunguyentrung) [inline app/views/layouts/_footer.html.erb]: Put style into scss file.

**Review by hieunguyentrung — CHANGES_REQUESTED** [review]

**Pull request merged**